docs/contributor-guide/developer-guide/new-full-time-swe-onboarding-materials.md

```diff
@@ -25,13 +34,14 @@
 
 Ensure your team lead grants you access to the following resources.
 
-* Azure
-* AWS
 * Slack
 * Added to the technical google group
-* LoginRadius
 * DockerHub (if at Google, need to pay for the professional one)
 * Github Access
   * [Civiform Org](https://github.com/civiform)
   * civiform/civiform
-  * civiform/docs 
+  * civiform/docs
+* Azure (optional, if working on deployment)
```

Comment:
Make the Deployment/optional part a header and then nest these 4 under it and remove the repetition.